src/main/java/org/isf/distype/rest/DiseaseTypeController.java

```diff
@@ -114,10 +114,10 @@ public ResponseEntity<DiseaseTypeDTO> updateDiseaseType(@Valid @RequestBody Dise
         if (!diseaseTypeManager.isCodePresent(diseaseType.getCode())) {
         	throw new OHAPIException(new OHExceptionMessage("Disease Type not found."), HttpStatus.INTERNAL_SERVER_ERROR);
         }
-        
-        if (diseaseTypeManager.updateDiseaseType(diseaseType)) {
-        	return ResponseEntity.ok(diseaseTypeDTO);
-        } else {
+        try {
+	        diseaseTypeManager.updateDiseaseType(diseaseType);
+	        return ResponseEntity.ok(diseaseTypeDTO);
+        } catch (OHServiceException serviceException) {
         	throw new OHAPIException(new OHExceptionMessage("Disease Type not updated."), HttpStatus.INTERNAL_SERVER_ERROR);
         }
 	}
```

Comment:
something wrong with the indentation at line 123 ?

Reply:
The original file had bad formatting and I just followed it.   I reformatted the method.